regex/parser/parser.go

```diff
@@ -247,18 +251,18 @@ func parseFile(rootParser *Parser, filename string, definitions map[string]strin
 // We removed flag merging because of https://github.com/coreruleset/crs-toolchain/issues/72
 func mergePrefixesSuffixes(target *Parser, source *Parser, out *bytes.Buffer) (*bytes.Buffer, error) {
 	logger.Trace().Msg("merging prefixes, suffixes from included file")
+	// If the included file has flags, this is an error
+	if len(source.Flags) > 0 {
+		return out, errors.New("include files must not contain flags. See https://github.com/coreruleset/crs-toolchain/issues/71")
+	}
```

Comment:
I think you are changing the semantics I've intended. Returning out instead of an empty newOut.